src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/advice/UserProfileConstants.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.userprofileapi.controller.advice;
+
+public class UserProfileConstants {
+
+    private UserProfileConstants() {
+    }
+
+    public static final String NAME_FORMAT_REGEX = "^[A-Za-z'-]+$";
```

Comment:
This RegEx will _only_ allow **aA - zZ** characters and only **'** and **-** as special characters